tests/test_sdist.py

```diff
@@ -30,3 +31,13 @@ def test_missing_dependency_format(resolve_dist, tmp_context):
     assert "setuptools>=40.8.0 -> 69.5.1" in s
     # Ensure we report what version we expect of all of the other dependencies
     assert "flit_core -> 3.9.0" in s
+
+@patch("fromager.sources.download_url")
+@patch("zipfile.ZipFile")
+def test_invalid_wheel_file_exception(mock_zipfile, mock_download_url):
+    mock_download_url.return_value = "fake_wheel.csv"
+    mock_zip_instance = MagicMock()
+    mock_zip_instance.namelist.return_value = ["file1.txt", "file2.txt"]
+    mock_zipfile.return_value = mock_zip_instance
+    sdist.download_wheel_check(["prebuilt1", "prebuilt2"], "http://example.com/fake_wheel.csv")
+    assertRaises(exceptions.TypeError, sdist.download_wheel_check)
```

Comment:
I think you want to be using `with pytest.raises()` here. There's an example in https://github.com/python-wheel-build/fromager/blob/main/tests/test_constraints.py#L25